.gitmodules

```
@@ -0,0 +1,3 @@
+[submodule "spec/fixtures/mauth-protocol-test-suite"]
+	path = spec/fixtures/mauth-protocol-test-suite
+	url = git@github.com:mdsol/mauth-protocol-test-suite.git
```

might need to change to `https`

```suggestion
	url = https://github.com/mdsol/mauth-protocol-test-suite.git
```

---

let me try it out

---

looks like that did it, thanks! CI breaking because there's no `Dir.children` in ruby < 2.5 i guess. will fix tmw

---

Ruby 2.3 and 2.4 have already reached EOL so I think we can just remove them:

https://github.com/mdsol/mauth-client-ruby/blob/a00161ecea7dcdb92cf66e9ebcda9ca1ac398984/.travis.yml#L5-L6

---

even better, removing

---

ah, we are still allowing them here...

https://github.com/mdsol/mauth-client-ruby/blob/a00161ecea7dcdb92cf66e9ebcda9ca1ac398984/mauth-client.gemspec#L14

@jfeltesse-mdsol probably we should keep supporting them, right?

---

I guess we can soft-deprecate, as in, the gemspec allows 2.3+ but we don't test < 2.5. No idea what version our customers may be using but if some are still on  < 2.5 we need to let the door open so they can use the v2 protocol if needs be.

---

But then, looking at the changes, we're dropping 2.3 and 2.4 just to replace that one line in the spec? Not worth the change IMHO. How about reverting that line, .travis.yml, the gemspec and call it a day?